test/caption-stream.test.js

```diff
@@ -815,6 +871,37 @@ QUnit.test('ignores CEA708 captions', function() {
   QUnit.equal(captions[2].text, 'WE TRY NOT TO PUT AN ANIMAL DOWN\nIF WE DON\'T HAVE TO.', 'parsed third caption correctly');
 });
 
+// Full character translation testa are below for Cea608Stream, they just only
+// test support for CC1
+QUnit.test('special and extended character codes work regardless of field and data channel', function() {
+  var packets, seiNals, captions = [];
+  packets = [
+    { ccData: 0x1c25, type: 0 },
+    { ccData: 0x1930, type: 0 },
+    { ccData: 0x1c2d, type: 0 },
+    { ccData: 0x1525, type: 1 },
+    { ccData: 0x2200, type: 1 },
+    { ccData: 0x123e, type: 1 },
+    { ccData: 0x152d, type: 1 },
+    { ccData: 0x1d25, type: 1 },
+    { ccData: 0x2200, type: 1 },
+    { ccData: 0x1a3f, type: 1 },
+    { ccData: 0x1d2d, type: 1 }
+  ];
+
+  captionStream.on('data', function(caption) {
+    captions.push(caption);
+  });
+
+  seiNals = packets.map(makeSeiFromCaptionPacket);
+  seiNals.forEach(captionStream.push, captionStream);
+  captionStream.flush();
+
+  QUnit.deepEqual(captions[0].text, String.fromCharCode(0xae), 'CC2 special character correct');
```

Comment:
Maybe add a comment with what the special/extended character is?

Reply:
Added more comments.

Reply:
One last thing that would be good to mention is that we expect `"` to be deleted because they come before an extended character

Reply:
I just said to see the tests below since it's explained there.